CMakeLists.txt

```diff
@@ -41,6 +39,17 @@ option(OTIO_AUTOMATIC_SUBMODULES "Fetch submodules automatically" ON)
 # Set option dependent variables
 
 if(OTIO_PYTHON_INSTALL)
+    # Find the python interpreter using FindPython module.
+    # The Development.Module component is used instead of the Development one
+    # because we don't need the libpython library since we don't link against it.
+    # pybind11 will detect that we already found a python
+    # interpreter and will use what was found previously (here).
+    # See https://pybind11.readthedocs.io/en/stable/compiling.html#findpython-mode
+    # Also, the "manylinux" docker images don't have libpython, so requiring the libraries is an issue there.
+    find_package(Python REQUIRED COMPONENTS Interpreter Development.Module)
```

Comment:
This block is in the section that runs regardless of whether the user has specified a specific python installation or not, is that correct?  I ask because the original line that was deleted was only in the branch where the user had not specified one.

Reply:
Exactly. The new behaviour is similar to the old one.

Before:
* If `OTIO_PYTHON_INSTALL_DIR` and `CMAKE_INSTALL_PREFIX` are not set, use `find_program(Python...)`
* Else don't call it explicitly and let pybind11 do the work of finding the interpreter later in the CMake file.

Now:
* No matter what happens, always use `find_program`. This should not change the previous behaviour as the `Python_*` variables created by `find_program(Python...)` are not used in the `else` block at lines 77-96.

Does that clarify things a little bit more @ssteinbach ?

Reply:
Yup, thanks!